presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -187,3 +187,46 @@ Accessors
 .. function:: ST_NumInteriorRing(Geometry) -> bigint
 
     Returns the cardinality of the collection of interior rings of a polygon.
+
+=====================
+Bing Tiles
```

Comment:
`Bing Tile Functions`?

Reply:
The title bars are too wide. They need to be the same width as the text.